Return the loaded library in torch.ops.load_library

842974287 opened the pull request:
Summary: ATT, return the loaded CDLL for further interaction.

Test Plan:
CI

Rollback Plan:

Differential Revision: D80011483

zou3519 commented on torch/_ops.py:
```diff
@@ -1477,17 +1477,21 @@ def load_library(self, path):
             except Exception as e:
                 raise OSError(f"Could not load this library: {path}") from e
         self.loaded_libraries.add(path)
+        return lib
```
what do you plan to do with this?

842974287 replied:
It's an internal use case, let me ping you in the workchat. But basically there are also extern "C" symbols that the program needs besides the torch custom ops. So `lib` would later used in `getattr(lib, "some_func_name")` to get the C function out in python.

albanD replied:
Why is calling CDLL again not a good solution here?

842974287 replied:
Mainly want to avoid introducing the logic from L1484 to the CDLL callsite to parse the internal buck path. But open to suggestions.

rec reviewed (approved):
Thanks for adding the types.

Given you need this value anyway, why not? It's certainly not unexpected that `load_library` would return a CDLL.

842974287 committed:
Return the loaded library in torch.ops.load_library (#160338)

Summary:

ATT, return the loaded CDLL for further interaction.

Test Plan:
CI

Rollback Plan:

Reviewed By: s4ayub

Differential Revision: D80011483

zou3519 reviewed (changes requested):
Sorry, I want to think about this some more. Please don't merge it yet.
